src/main/java/com/pikachu/purple/infrastructure/persistence/review/entity/ReviewJpaEntity.java

```diff
@@ -0,0 +1,34 @@
+package com.pikachu.purple.infrastructure.persistence.review.entity;
+
+import com.pikachu.purple.infrastructure.persistence.common.BaseEntity;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@Entity
+@Table(name = "review")
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class ReviewJpaEntity extends BaseEntity {
+
+    @Id
+    @Column(name = "review_id")
+    private Long reviewId;
+
+    @Column(name = "perfume_id", nullable = false)
+    private Long perfumeId;
+
+    @Column(name = "user_id", nullable = false)
+    private Long userId;
+
+    @Column(name = "rating_id", nullable = false)
+    private Long ratingId;
+
+    @Column(name = "comment", nullable = false)
```

Comment:
p3) 해당 컬럼은 무슨 역할인가요? 만약 리뷰 내용이라면 content 정도가 적절해 보이는 데 어떨까요??
```suggestion
    @Column(name = "comment", nullable = false)
```